modules/tinymce/src/themes/silver/test/ts/headless/components/imagepanel/BasicImagePanelTest.ts

```diff
@@ -0,0 +1,117 @@
+import { ApproxStructure, Assertions, UiFinder } from '@ephox/agar';
+import { AlloyComponent, GuiFactory, Representing, TestHelpers } from '@ephox/alloy';
+import { describe, it } from '@ephox/bedrock-client';
+import { Optional } from '@ephox/katamari';
+import { SugarElement } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import { ImagePanelData, renderImagePanel } from '../../../../../main/ts/ui/dialog/ImagePanel';
+
+// Dupe from the image panel. This definitely belongs somewhere else.
+const loadImage = (image: SugarElement<HTMLImageElement>): Promise<SugarElement<HTMLImageElement>> => new Promise((resolve) => {
+  const loaded = () => {
+    image.dom.removeEventListener('load', loaded);
+    resolve(image);
+  };
+
+  if (image.dom.complete) {
+    resolve(image);
+  } else {
+    image.dom.addEventListener('load', loaded);
+  }
+});
+
+describe('headless.tinymce.themes.silver.components.imagepanel.Image Panel component Test', () => {
```

Comment:
```suggestion
describe('headless.tinymce.themes.silver.components.imagepanel.BasicImagePanelTest', () => {
```
Also this tests filename is inconsistent with the rest... it probably should just be called `ImagePanelTest` or similar.

Reply:
This was literally copy/paste from another test file, I assumed if it was good enough already it was good enough here.
https://github.com/tinymce/tinymce/blob/3ec7261b9f24c22aaae6a1ddc0ffa47cd8db5ab2/modules/tinymce/src/themes/silver/test/ts/headless/components/checkbox/BasicCheckboxTest.ts#L10

which is also why it didn't end up with a JIRA tag.

Reply:
Fair... ignore then as we obviously need to clean this up :disappointed:

Reply:
I fixed the describe string - I did think it was weird that the name didn't match - and left the filename.